packaging/hudi-presto-bundle/pom.xml

```diff
@@ -128,5 +133,18 @@
       <artifactId>hudi-hadoop-mr-bundle</artifactId>
       <version>${project.version}</version>
     </dependency>
+
+    <!-- Parquet -->
+    <dependency>
+      <groupId>org.apache.parquet</groupId>
```

Comment:
@bschell where are we using this dependency ?

Reply:
These are used by the HoodieParquetRealtimeRecordReader that is now being used for presto MOR realtime queries. The record reader will fail if these dependencies are there.